framework/src/vectorpostprocessors/PointSamplerBase.C

```diff
@@ -100,7 +108,7 @@ void
 PointSamplerBase::finalize()
 {
   // Save off for speed
-  unsigned int pid = processor_id();
+  const auto pid = processor_id();
```

Comment:
Much better... we gotta get these `unsigned int`s out of the code... and, yes, I realize that a lot of them are my fault...